analyzer/codechecker_analyzer/buildlog/build_manager.py

```diff
@@ -63,6 +66,16 @@ def perform_build_command(logfile, command, context, keep_link, silent=False):
         with open(original_env_file, 'rb') as env_file:
             original_env = pickle.load(env_file)
 
+        # The env is loaded we do not need it anymore.
```

Comment:
We will remove this directory at the end of the main CodeChecker script too:
https://github.com/Ericsson/codechecker/blob/8fdc99772c66950fc1c5c2c0ffcaec27585666ac/bin/CodeChecker#L82-L90

Why do we need to remove it in both places?

Reply:
in my other PR #2074 this will be removed, probably it is better if I make this change in that PR and not here